fixtures/packaging/babel-standalone/dev.html

```diff
@@ -5,10 +5,8 @@
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
     <div id="container"></div>
     <script type="text/babel">
-      ReactDOM.render(
-        <h1>Hello World!</h1>,
-        document.getElementById('container')
-      );
+      ReactDOM.createRoot( document.getElementById('container')).render(
+      <h1>Hello World!</h1>)
```

Comment:
```suggestion
      ReactDOM.createRoot(document.getElementById("container")).render(
        <h1>Hello World!</h1>
      );
```

Reply:
Why did the suggested formatting not work? It's in line with what Prettier outputs.

Reply:
seems like my prettier config tries to add colons to the end of the file! fixed it and ran prettier

Reply:
@eps1lon sorry for the wrong push! fixed the prettier error, this is ready to go now!